src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```
@@ -9,6 +9,20 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
+        User tmp = storageDao.get(user.getLogin());
+        if (user.getAge() == null || user.getLogin() == null || user.getPassword() == null) {
+            throw new NullPointerException();
```

Issue: Throwing `NullPointerException` for invalid input is not appropriate. According to standard Java practices and the checklist, you should throw `IllegalArgumentException` or a custom exception for invalid arguments. Please update this to throw the correct exception type.